### PR TITLE
fix(detail page): Changes description placeholder to 'Add a Description'. Fixes #1602.

### LINKED ISSUE #1602: Incorrect Placeholder Text
The Description field has an incorrect set of placeholder text. Instead of 'Edit Text', it should read 'Add a Description'.

screen shot 2017-04-12 at 7 14 16 am

### PR DESCRIPTION
**Pull request by nimishamukherjee**
fix(detail page): Changes description placeholder to 'Add a Description'. Fixes #1602.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1650-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1650-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1650-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1650-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1650-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1650-fabric8-planner.int.rdu2c.fabric8.io

File: src/app/work-item/work-item-detail/markdown-control/markdown-control.component.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ constructor(
     this.textEditable = false;
     this.showPreview();
     // emit save event
-    this.onUpdate.emit(this.markdownText.trim());
+    this.onUpdate.emit(this.markdownText.trim())
```
Comment (vikram-raj) [inline src/app/work-item/work-item-detail/markdown-control/markdown-control.component.ts]: Semicolon missing.

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1650-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1650-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1650-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1650-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1650-fabric8-planner.int.rdu2c.fabric8.io

**Pull request merged**